cmd/producer/main.go

```diff
@@ -58,15 +69,15 @@ type publisherCloser interface {
 }
 
 func main() {
-	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, os.Kill)
```

Comment:
Is there a specific reason for switching from `syscall` to `os`?
I had previously updated it to use `syscall` https://github.com/rudderlabs/rudder-load/commit/cf6d60b11cba7a83ba3fb95ae7a4380554aaf5c9 because we were facing issues in handling `SIGTERM`, which caused requests failing with a 499 status.

Reply:
Compatibility depending on OS since we might run these script on the devs machines I thought I'd use the `os` package instead. I wasn't aware of the issue you came across to though. I'll add back `syscall.SIGTERM` as well then so effectively we'll support `SIGINT`, `SIGKILL` and `SIGTERM` :+1: